lib/bindings/src/breez_sdk_liquid.udl

```diff
@@ -325,10 +325,15 @@ enum PaymentError {
     "SignerError",
 };
 
+[Enum]
+interface BlockchainExplorer {
+    Electrum(string url);
+    Esplora(string url, boolean use_waterfalls);
+};
+
 dictionary Config {
-    string liquid_electrum_url;
-    string bitcoin_electrum_url;
-    string mempoolspace_url;
+    sequence<BlockchainExplorer> liquid_explorers;
+    sequence<BlockchainExplorer> bitcoin_explorers;
```

Comment:
I suggest to start simple without redundancy. one explorer for bitcoin, one for liquid.